networkit/helpers.pxd

```diff
@@ -27,3 +27,7 @@ cdef extern from "<algorithm>" namespace "std":
 	pair[_Graph, vector[node]] move(pair[_Graph, vector[node]]) nogil
 	vector[pair[pair[node, node], double]] move(vector[pair[pair[node, node], double]]) nogil
 	vector[pair[node, node]] move(vector[pair[node, node]]) nogil
+
+#cdef extern from "<networkit/Globals.hpp>" namespace "NetworKit":
+
+#	index _none "NetworKit::none"
```

Comment:
?

Reply:
This was unnecessary, but is resolved in a later commit.

Reply:
This could be removed using `rebase -i`.